code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -58,6 +58,21 @@
 				//End bloody footprints
 
 				S.step_action()
+		
+	if(bleed_rate && !buckled && has_gravity(loc) && lying)
+		if(prob(2) && istype(loc, /obj/structure/closet))
+			visible_message("<span class='warning'>[src] bumps their head against \the [src]!</span>")

```

Comment:
You can only bump your head if you're bleeding?

Reply:
If traumas are merged this should totally have a chance of concussing him